devices.js

```diff
@@ -5374,7 +5374,25 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
-
+    {
+        zigbeeModel: ['LXN56-1S27LX1.2'],
+        model: 'NUE-ZBFLB',
+        vendor: 'Nue / 3A',
+        description: 'Smart Fan Light Switch',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'light': 1, 'fan_high': 2, 'fan_med': 3, 'fan_low': 4};
```

Comment:
can you change this to `return {'l1': 1, 'l2': 2, 'l3': 3, 'l4': 4};` and control it via `zigbee2mqtt/Hannah Light Fan Switch/l1/set` payload `{"state": "ON"}`, does this work?